workflow/controller/controller.go

```diff
@@ -991,3 +1003,12 @@ func (wfc *WorkflowController) syncWorkflowPhaseMetrics() {
 		wfc.metrics.SetWorkflowPhaseGauge(phase, len(objs))
 	}
 }
+
+func (wfc *WorkflowController) syncPodPhaseMetrics() {
+	objs, err := wfc.podInformer.GetIndexer().ByIndex(indexes.PodPhaseIndex, string(apiv1.PodRunning))
```

Comment:
I'd like gauges for all pod phases please

Reply:
Done. Only added `PodRunning` and `PodPending` as these are the only phases which the complete-sensitive Pod informer will still report about